content/docs/addons-shallow-renderer.md

```diff
@@ -13,11 +13,11 @@ import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
 var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
 ```
 
-## Overview {#overview}
+## ओवरव्यू {#overview}
 
-When writing unit tests for React, shallow rendering can be helpful. Shallow rendering lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+React के लिए यूनिट टेस्ट लिखते समय शैलो रेंडरिंग काफ़ी उपयोगी हो सकती है। शैलो रेंडरिंग आपको एक कौम्पोनॅन्ट को "एक स्तर गहरा" रेंडर करने देता है और आपको दृढ़ता से ये जांच करने की क्षमता प्रदान करता है कि इस कौम्पोनॅन्ट का रेंडर मेथड क्या return करेगा, child कौम्पोनॅन्टस के व्यव्हार की चिंता किये बिना, जो की instantiate या रेंडर नहीं होते। इसमें DOM की आवश्यकता नहीं होती है।
```

Comment:
दृढ़ता could be replaced by an easier to understand word?

Reply:
@saranshkataria I am thinking to work on this. And we will get this merged.
How about ` वास्तविकता`?

Reply:
`आपको दृढ़ता से ये जांच करने की क्षमता प्रदान करता है` => `और रेंडर मेथड जो रिटर्न कर रहा है उसको अस्सर्ट कर सकते हैं`

Reply:
sounds good